### DISCUSSION TITLE
I would like to implement the syntax highlight for SQL

### DISCUSSION THREAD

**Discussion by eduardonunesp** [zed-industries/zed#7083] [Show and tell]
Hi everyone, well I've been kind of hyped with this project, I've been using this week and I notice no support for SQL, so I started to implement myself. But I'm not sure if I'm doing this right or not, at least right now the syntax on SQL files are not coloring, but at least thy type is been correctly assigned when open SQL files.

<img width="715" alt="image" src="https://github.com/zed-industries/zed/assets/582516/ccf198b3-a8da-4f70-bb7c-de7166610139">

Can some give me a light here how to achieve this highlight stuff, also, would be nice some guidance to understand the `.scm` files :D, Thanks in advance, this the PR https://github.com/zed-industries/zed/pull/7082

**Comment by eduardonunesp** [comment]
Now it's working !

<img width="448" alt="image" src="https://github.com/zed-industries/zed/assets/582516/20fc28ce-db83-4c7a-a857-0105d97a7704">

**Comment by SomeoneToIgnore** [comment]
Since it's moving somewhere, and we have a standalone issue and now PR for that, this one seems good to be closed.
